__tests__/app.test.js

```diff
@@ -34,3 +35,23 @@ describe("GET/api/topics", () => {
       });
   });
 });
+
+describe("GET/api", () => {
+  test("status: 200, should respond with a json file describing each endpoint", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual(endpoints);
+      });
```

Comment:
You should be more specific about what form this JSON file should take. Maybe have a look at .toMatchObject( ) and expect.objectContaining( ) 👀